shared/provision/set-public-name/index.tsx

```diff
@@ -32,10 +33,18 @@ const SetPublicName = (props: Props) => {
         },
       ]}
       onBack={props.onBack}
-      title={`Name this ${isMobile ? 'phone' : 'computer'}`}
+      title={`Name this ${Styles.isMobile ? 'phone' : 'computer'}`}
     >
       <Kb.Box2 direction="vertical" style={styles.contents} centerChildren={true} gap="medium">
-        <Kb.Icon type={isMobile ? 'icon-phone-96' : 'icon-computer-96'} />
+        <Kb.Icon
+          type={
+            Styles.isMobile
+              ? Platform.isLargeScreen
+                ? 'icon-phone-background-1-96'
```

Comment:
We should be able to get the actual next device icon to show the right icon instead of always icon 1 here - lmk if you want me to give you a rundown on how the device icon numbering stuff works; it's kinda funky.